packages/vuetify/src/components/VBtn/VBtn.ts

```diff
@@ -81,16 +82,19 @@ export default baseMixins.extend<options>().extend({
         'v-btn--absolute': this.absolute,
         'v-btn--block': this.block,
         'v-btn--bottom': this.bottom,
-        'v-btn--contained': this.contained,
-        'v-btn--depressed': (this.depressed) || this.outlined,
+        'v-btn--contained': this.isElevated, // TODO: remove v3
```

Comment:
"isElevated" looks different from other names?  (should prefix is?)

Reply:
It looks like it's a boolean. If the other properties are booleans should we adopt the 'is' syntax for future updates?

Reply:
things like `depressed` and `outlined` are public style props. `isElevated` is an internal computed prop and is in line with other computed props such as `isRound`